troi/musicbrainz/artist_credit_id_lookup.py

```diff
@@ -38,12 +38,12 @@ def read(self, inputs, debug=False):
 
         r = requests.get(url)
         if r.status_code != 200:
-            r.raise_for_status()
+            raise PipelineError("Cannot fetch artist credits from ListenBrainz: HTTP code %d" % r.status_code)
```

Comment:
musicbrainz? Or is this ListenBrainz because of the API that it's hitting?

Reply:
The latter -- its hitting an LB api, this error message LB.